whats_fresh/whats_fresh_api/templates/vendor.html

```diff
@@ -48,15 +48,23 @@ <h4 id='error_tag' class='light warning'></h4>
         {{ vendor_form.story }}
         <p class=" field_text">In Port:</p>
         {{ vendor_form.status }}
-        <h3>Location</h3>
-        <p class="field_text">Street Address*:</p>
-        {{ vendor_form.street }}
-        <p class="field_text">City*:</p>
-        {{ vendor_form.city }}
-        <p class="field_text">State*:</p>
-        {{ vendor_form.state }}
-        <p class="field_text">Zipcode*:</p>
-        {{ vendor_form.zip }}
+        <h3>Location</h3><br>
+
+    <!-- Code for Map based location inputs start -->
+
+    <script type="text/javascript" src='http://maps.google.com/maps/api/js?sensor=false&libraries=places'></script>
+
+    <div><input type="text" id="us3-address" onkeypress="removerEnter(event)"/></div>
+
+    <div id="us3" class="map"></div><br>
+
+                <p class="field_text">Latitude*:</p>
+                <div><input type="text" id="us3-lat" onkeypress="removerEnter(event)" enctype="multipart/form-data" value="{{ vendor_form.latitude }}" /></div>
```

Comment:
I believe that the id of the input field is what's sent as the name of the field, so instead of us3-lat and us3-lon, these should be latitude and longitude. I don't know how easy this would be to do?